packages/vuetify/src/components/VTooltip/__tests__/VTooltip.spec.cy.tsx

```diff
@@ -0,0 +1,39 @@
+/// <reference types="../../../../types/cypress" />
+
+// Components
+import { VTooltip } from '..'
+
+// Utilities
+const colors = ['success', 'info', 'warning', 'error', 'invalid']
```

Comment:
Besides of these, can it support hex code?

Reply:
Yes, it works with hex codes too, see the markup example. I can add some hex codes to the test if you like.